[2024-06-25T16:38:48.760+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T16:38:48.779+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_neo_data_to_s3.fetch_nasa_data scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T16:38:48.785+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_neo_data_to_s3.fetch_nasa_data scheduled__2024-06-24T00:00:00+00:00 [queued]>
[2024-06-25T16:38:48.785+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-06-25T16:38:48.796+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_nasa_data> on 2024-06-24 00:00:00+00:00
[2024-06-25T16:38:48.802+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=69) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T16:38:48.803+0000] {standard_task_runner.py:63} INFO - Started process 73 to run task
[2024-06-25T16:38:48.803+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_neo_data_to_s3', 'fetch_nasa_data', 'scheduled__2024-06-24T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/etl_neo.py', '--cfg-path', '/tmp/tmp5xzzajaa']
[2024-06-25T16:38:48.805+0000] {standard_task_runner.py:91} INFO - Job 75: Subtask fetch_nasa_data
[2024-06-25T16:38:48.842+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_neo_data_to_s3.fetch_nasa_data scheduled__2024-06-24T00:00:00+00:00 [running]> on host 95b3b965c5b1
[2024-06-25T16:38:48.904+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_neo_data_to_s3' AIRFLOW_CTX_TASK_ID='fetch_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T00:00:00+00:00'
[2024-06-25T16:38:48.905+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T16:38:49.981+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-25T16:38:49.982+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T16:38:49.991+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_neo_data_to_s3, task_id=fetch_nasa_data, run_id=scheduled__2024-06-24T00:00:00+00:00, execution_date=20240624T000000, start_date=20240625T163848, end_date=20240625T163849
[2024-06-25T16:38:50.020+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T16:38:50.040+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-25T16:38:50.041+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
